source/mbed_platform.cpp

```diff
@@ -61,8 +69,24 @@ uint32_t getTimeOverflows(){
 }
 
 void sleepFromUntil(tick_t now, tick_t until){
+
+#if YOTTA_CFG_MINAR_TEST_CLOCK_OVERFLOW
+    MBED_HOSTTEST_ASSERT(now <= Time_Mask && until <= Time_Mask);
```

Comment:
This is a bit weird. The code in this particular file doesn't really use the mbed testing framework, so this line assumes that the function is called from some code that definitely uses the testing framework somehow, and that is was properly initialized and so on. It doesn't feel like the right thing to do here.